domain_model/Election.ts

```diff
@@ -20,13 +20,15 @@ export interface Election {
     settings:       ElectionSettings;
     auth_key?:      string;
     claim_key_hash?: string;
-    is_public?:      Boolean;
-    create_date:    Date | string;
```

Comment:
Does Date | string mean there's multiple ways this can be stored in the DB? 

I would be cleaner if the format could be consistent, but I have no idea how much work that would be so feel free to ignore

Reply:
The reason for it is that dates get converted to string when converted to JSON. I think they're only being stored as date objects in the front end to interface with the date logic better.